# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nullmailer/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Get the tempalte friends to compile with gcc-3.4.

  - Rene Rebe <rene@exactcode.de

--- ./lib/list.h.vanilla	2004-07-05 13:54:18.000000000 +0200
+++ ./lib/list.h	2004-07-05 14:01:49.000000000 +0200
@@ -9,18 +9,19 @@
   ~list_node() { }
 };
 
-template<class T> class list_iterator;
-template<class T> class const_list_iterator;
+template<typename T> class list_iterator;
+template<typename T> class const_list_iterator;
 
-template<class T> class list
+template<typename T> class list
 {
 public:
   typedef list_node<T> node;
   typedef list_iterator<T> iter;
   typedef const_list_iterator<T> const_iter;
-  friend class iter;
-  friend class const_iter;
-  
+
+  friend class list_iterator<T>;
+  friend class const_list_iterator<T>;
+ 
   list()
     : head(0), tail(0), cnt(0)
     {
